workers/unity/Packages/io.improbable.gdk.tools/LocalLaunch.cs

```diff
@@ -38,7 +40,15 @@ private static void BuildConfigMenu()
         private static void LaunchMenu()
         {
             Debug.Log("Launching SpatialOS locally...");
-            EditorApplication.delayCall += LaunchLocalDeployment;
+            EditorApplication.delayCall += ConfigureLocalDeployment;
+        }
+
+        [MenuItem("SpatialOS/Local launch with custom snapshot", false, MenuPriorities.LocalLaunch)]
+        private static void LaunchWithSnapshotMenu()
+        {
+            gdkToolsConfig = GdkToolsConfiguration.GetOrCreateInstance();
```

Comment:
There is no need to store this globally just to print the path name.

Reply:
the path name is used in the `spatial local launch` call when launching the local deployment, but I'll move it to a playerpref